newswires/app/db/FingerpostWireEntry.scala

```diff
@@ -1,27 +1,59 @@
 package db
 
-import play.api.libs.json.{Json, OFormat, OWrites}
+import play.api.libs.json._
 import scalikejdbc._
 
 import java.time.ZonedDateTime
 
+case class FingerpostWireSubjects(
+    code: Seq[String]
+)
+object FingerpostWireSubjects {
+  implicit val format: OFormat[FingerpostWireSubjects] =
+    Json.format[FingerpostWireSubjects]
+}
+
 case class FingerpostWire(
     uri: Option[String],
+    sourceFeed: Option[String],
     usn: Option[String],
     version: Option[String],
+    status: Option[String],
     firstVersion: Option[String],
     versionCreated: Option[String],
     dateTimeSent: Option[String],
+    slug: Option[String],
     headline: Option[String],
     subhead: Option[String],
     byline: Option[String],
+    priority: Option[String],
+    subjects: Option[FingerpostWireSubjects],
     keywords: Option[List[String]],
+    language: Option[String],
     usage: Option[String],
     location: Option[String],
-    body_text: Option[String]
+    bodyText: Option[String]
 )
 object FingerpostWire {
-  implicit val format: OFormat[FingerpostWire] = Json.format[FingerpostWire]
+  // rename a couple of fields
+  private val reads: Reads[FingerpostWire] =
+    Json.reads[FingerpostWire].preprocess { case JsObject(underlying) =>
+      JsObject(underlying.map {
+        case ("source-feed", v) => ("sourceFeed", v)
+        case ("body_text", v)   => ("bodyText", v)
+        case pair               => pair
+      })
+    }
+  //// more play-jsony alternative:
```

Comment:
fwiw I prefer the version that you've got -- feels more readable to me at least

should we remove the commented code or do you think it's useful for documentation?

Reply:
Thanks - if you agree with the uncommented version then that's good enough for me - the alternative has been removed